WordPress/Sniffs/VIP/SlowDBQuerySniff.php

```diff
@@ -50,6 +51,10 @@ public function getGroups() {
 	 */
 	public function process_token( $stackPtr ) {
 
+		if ( $this->has_whitelist_comment( 'slow query', $stackPtr ) ) {
+			return;
+		}
+
 		if ( $this->has_whitelist_comment( 'tax_query', $stackPtr ) ) {
 			return;
```

Comment:
If it helps, I can run a quick search of the plugin repo to see how often the whitelist comment is used within those 50.000+ plugins to get some indication. (won't be able to run this until Wednesday though, still in Rome at the moment)